web/modules/custom/weather/src/Plugin/Block/WeatherBlock.php

```diff
@@ -151,4 +218,76 @@ private function getLocation() {
     return $this->config->get('weather.default_city') ?? 'Lutsk, Ukraine';
```

Comment:
It is better to return NULL than use the hardcoded value.
Of course, you have to handle it in the place where the method is called.